fendermint/fendermint/vm/interpreter/src/fvm/state/ipc.rs

```diff
@@ -31,12 +30,29 @@ use super::{
 use crate::fvm::FvmApplyRet;
 use fendermint_vm_actor_interface::ipc;
 use fvm_shared::econ::TokenAmount;
+use ipc_actors_abis::checkpointing_facet::CheckpointingFacet;
+use ipc_actors_abis::top_down_finality_facet::TopDownFinalityFacet;
+use ipc_actors_abis::xnet_messaging_facet::XnetMessagingFacet;
 
 #[derive(Clone)]
 pub struct GatewayCaller<DB> {
     addr: EthAddress,
     getter: ContractCaller<DB, GatewayGetterFacet<MockProvider>, NoRevert>,
-    router: ContractCaller<DB, GatewayRouterFacet<MockProvider>, router::GatewayRouterFacetErrors>,
+    checkpoint_router: ContractCaller<
+        DB,
+        CheckpointingFacet<MockProvider>,
+        checkpointing_facet::CheckpointingFacetErrors,
+    >,
+    topdown_router: ContractCaller<
+        DB,
+        TopDownFinalityFacet<MockProvider>,
+        top_down_finality_facet::TopDownFinalityFacetErrors,
+    >,
+    xnet_messaging_facet: ContractCaller<
+        DB,
+        XnetMessagingFacet<MockProvider>,
+        xnet_messaging_facet::XnetMessagingFacetErrors,
+    >,
```

Comment:
The naming convention was `WhateverFacet` became `whatever` field, so I think these should be called `checkpointing`, `topdown` and `xnet_messaging`. I can't see why it's `checkpoint_router` as "router" does not appear in the type, and all of them are facets so `_facet` in the third is redundant.